Sources/Client/Extensions/Codable+Extensions.swift

```diff
@@ -166,8 +166,8 @@ extension Encodable {
     }
 }
 
-/// Try in the most cases map it to `Void`.
+/// Empty `Codable` data.
 public struct EmptyData: Codable {
-    /// A shared instance of empty data.
+    /// A shared instance of empty `Codable` data.
```

Comment:
1. I made it class.
2. I found one place where it was used as `EmptyData()` in `Endpoint`.

Reply:
But why do we need a shared instance of `EmptyData`?

Reply:
I missed understanding you. Ofc it's better to leave it as a struct.
So, do you prefer to use `EmptyData()`?

Reply:
In this case, we should make a public `init`, because by default will be available only init with a decoder. So, I think it's handier to have `.empty`

Reply:
I wasn't suggesting something :) I really just asked why we need a shared instance of `EmptyData`.

Reply:
It needs external frameworks like Core.

Reply:
I think we don't understand each other. Let's leave it as it, it's just a detail.

Reply:
It's actually not shared. A type for empty decodable data.